trace/cloud-trace-demo-app-opentelemetry/app/Dockerfile

```diff
@@ -16,12 +16,14 @@
 # https://hub.docker.com/_/python
 FROM python:3.10-slim
 
-# Copy local code to the container image.
+RUN pip3 install --upgrade pip
+
 ENV APP_HOME /app
 WORKDIR $APP_HOME
-COPY . ./
-COPY requirements.txt /app/
-RUN pip install Flask gunicorn
+
```

Comment:
I meant only `apt-get update` isn't necessary, and you can clean up `apt` repository to make the container lighter.

Reply:
I understand. My thought was that since I did not use it, there is no reason to clean it. :slightly_smiling_face: